### DISCUSSION TITLE
v3.4.x

### DISCUSSION THREAD

**Discussion by wjbenfold** [SciTools/iris#4433] [Releases]
The **release manager** for `v3.4.x` is @trexfeathers 

The **release buddy** for `v3.4.x` is @stephenworsley  

Release branch: [`v3.4.x`](https://github.com/SciTools/iris/tree/v3.4.x)

| Minor Release | Status | Project | Date | RTD | Release | conda-forge | PyPI |
| :---: | :---: | :---: | :---: | :---: | :---: | :---: | :---: |
| `v3.4.0rc0` | Released | [Iris `v3.4.0`](https://github.com/orgs/SciTools/projects/3) | `2022-11-17` | [![RTD](https://img.shields.io/badge/🔒_version_(archived)-3.4.0rc0-red?style=?style=flat)](https://scitools-iris.readthedocs.io/en/v3.4.0rc0/) | [v3.4.0rc0](https://github.com/SciTools/iris/releases/tag/v3.4.0rc0) | [![Anaconda-Server Badge](https://img.shields.io/badge/Anaconda.org-3.4.0rc0-blue.svg)](https://anaconda.org/conda-forge/iris/files?version=3.4.0rc0) | [![PyPI](https://img.shields.io/badge/PyPI-3.4.0rc0-orange.svg)](https://pypi.org/project/scitools-iris/3.4.0rc0/) |
| `v3.4.0` | Released | [Iris `v3.4.0`](https://github.com/orgs/SciTools/projects/3) | `2022-12-01` | [![RTD](https://img.shields.io/badge/🔒_version_(archived)-3.4.0-red?style=?style=flat)](https://scitools-iris.readthedocs.io/en/v3.4.0/) | [v3.4.0](https://github.com/SciTools/iris/releases/tag/v3.4.0) | [![Anaconda-Server Badge](https://img.shields.io/badge/Anaconda.org-3.4.0-blue.svg)](https://anaconda.org/conda-forge/iris/files?version=3.4.0) | [![PyPI](https://img.shields.io/badge/PyPI-3.4.0-orange.svg)](https://pypi.org/project/scitools-iris/3.4.0/) |
| `v3.4.1` | Released | [Iris `v3.4.0`](https://github.com/orgs/SciTools/projects/3) | `2023-02-21` | [![RTD](https://img.shields.io/badge/🔒_version_(archived)-3.4.1-red?style=?style=flat)](https://scitools-iris.readthedocs.io/en/v3.4.1/) | [v3.4.1](https://github.com/SciTools/iris/releases/tag/v3.4.1) | [![Anaconda-Server Badge](https://img.shields.io/badge/Anaconda.org-3.4.1-blue.svg)](https://anaconda.org/conda-forge/iris/files?version=3.4.1) | [![PyPI](https://img.shields.io/badge/PyPI-3.4.1-orange.svg)](https://pypi.org/project/scitools-iris/3.4.1/) |

**Comment by trexfeathers** [comment]
Take note! Iris `v3.4.0` is scheduled for `2022-12-01`.

That should hopefully allow testing time for ESMValTool `v.2.8.0` (📢 @bouweandela @zklaus @schlunma).

**Comment by trexfeathers** [comment]
Hopefully you've all seen by now that `v3.4.0rc0` is out!

**Comment by trexfeathers** [comment]
`v3.4.0` has now been released 🙂

**Comment by trexfeathers** [comment]
`v3.4.1` has been released, which makes NetCDF file handling thread-safe, thus removing our NetCDF4 pin.
